### PR TITLE
Smashing Magazine clone

### PR DESCRIPTION
**Pull request by marcelomaidden**
## Smashing Magazine clone

What we did on this project

- Implemented navbar using grid
- Used font awesome for icons
- Used flex and grid on sections
- Used float to position elements
- Used filter: grayscale function to change the images to gray shades
- Used gray heat map on the entire page

### CODE REVIEW

File: index.html
Comment (teekaytech) [inline index.html]: The value of the `alt` attribute is missing. Please run through your code and give the value of the `alt` attribute in this line and subsequent ones.

File: index.html
Comment (teekaytech) [inline index.html]: These list items should be linked individually with `<a>` tag.

File: index.html
Comment (teekaytech) [inline index.html]: You have to link these ones too.

File: stylesheets/main.css
Comment (teekaytech) [inline stylesheets/main.css]: The rules specified in these blocks are similar. Please keep your code DRY by grouping similar rules together.



[Here](https://speckyboy.com/good-bad-css-practices) is the link to the article referenced in that screenshot.

**Review by teekaytech — CHANGES_REQUESTED** [review]
## Status: Changes Required :warning: :vertical_traffic_light: 

### Hello @marcelomaidden & @che30 :wave:

__I must say, you've done an amazing job so far.__
- Your nice readme file :ok_hand: 
- Well-arranged page layout :+1: 
- Descriptive PR title and summary :ok:  
- Your linters are green :100: 
- Nice use of grid and flex :cool: 

__Overall, this is an impressive work. We just have to make some changes that are required before this project can be approved. Let's dive in:__

**[ REQUIRED CHANGES ]**

- [ ] We are having a directory `.vscode` in our project. I believe this is not related to the project. Things like this can be taken care of by the `.gitignore` file. [This article](https://docs.github.com/en/free-pro-team@latest/github/using-git/ignoring-files#create-a-local-gitignore) talks about using the `.gitignore` file to exclude files that we do not want to push to GitHub.



- [ ] We're having some color other than greyscale in our project. Let's change these colors to any shades of gray(white, black, etc) we wish. 



- [ ] We have to link our footer items as I've mentioned in my comments below. Let's fix that too.





**COMMENTS**
- I observed that you made a set of commits while trying to fix linter errors towards the end. I personally think we can be more professional about this by setting up our linters locally and fix the errors reported before committing. If you need help with setting up the linters locally, you can check out the second part of [this article](https://github.com/microverseinc/linters-config/tree/master/html-css)

**That's it! Please make the required changes and submit another review request when you are done.**

* * *

**_You can reach out to me on twitter (@ola_lere), Github(@teekaytech) or slack (@taofeekolalere )_**
__Happy Coding :smiley:__

**Review by teekaytech — CHANGES_REQUESTED** [review]
## Status: Changes Required :warning: :vertical_traffic_light: 

### Hello team :wave:

**You've done well in fixing those changes highlighted earlier. One more change is still not fixed though. Let's break it down.**

**[ REQUIRED CHANGES ]**

- [ ] The `.vscode` folder is still present in your project. 
- I can see you've added the `.gitignore` file. You should note that the `.gitignore` file will not remove a file/folder that has already been committed.
- Since this is the case, let's find another way to remove the folder.
  - Open your feature branch on your local PC.
  - Delete the `.vscode` folder manually from your project.
  - Stage and commit your changes.
  - Push the committed changes. The remote copy of the folder should be removed by then.

- [ ] Since you've made a couple of changes in your project, you should not forget to update your live demo link.

**Please make the required changes and submit another review request when you are done.**

* * *

**_You can reach out to me on twitter (@ola_lere), Github(@teekaytech) or slack (@taofeekolalere )_**
__Happy Coding :smiley:__

**Review by javitocor — CHANGES_REQUESTED** [review]
# STATUS CHANGES REQUIRED ♻️ 

Good effort with the changes made so far, team!

But we still have some things before approving this project and move forward 💪 
Kindly, check my comments below 👇  apply that changes and resubmit your project for a new code review ⏩ 

## Changes

- We still have one `background: linear-gradient(RGB(131, 98, 98), #686868);` colour that is not belonging to the greyscale, please make all the colours of the page in grayscale.
.- In this project the content is not important, what we are doing here is to create a heatmap of the page to understand better the importance of each element in the page layout, we want to know which elements catch more our attention and which ones are not important. You have done a great effort cloning the page, but this is not what the project is about. Check [this](https://web.archive.org/web/20170628134444/http://www.vanseodesign.com/blog/wp-content/uploads/2009/12/visual-hierarchy-compared.png) image from the Odin Project to understand what I am trying to say. You have added photos and images which makes that I cannot see the importance of these elements in the layout of the page. For example, the 'seriously, red?' element in the main section, you painted white, which is the lowest colour in the greyscale, and this element highlights and catch your attention when you first load the page. Please modify the different elements of the page into an appropriate colour. You can use [this](https://chrome.google.com/webstore/detail/grayscale-the-web-save-si/mblmpdpfppogibmoobibfannckeeleag?hl=en) google extension and compare it with your site.
What I am asking you is to highlight the elements in the proper colour, create a heatmap, not a clone. One more example, the links in the navbar, there are some more important than others, but in your site are all equal.

**Review by teekaytech — APPROVED** [review]
# Status: Approved :green_circle: 


### Congratulations team, this project has passed all the mandatory requirements :tada: It is therefore approved :heavy_check_mark:

**Comment(s)**
- Before you merge to master, kindly update the project screenshot in your README file.

* * * 
**Again, congratulations! You can merge to master.
Do not forget to fill the project completion form.
Good luck with your next project.**

**I'm @taofeekolalere on slack.
Happy Coding :man_technologist:**

**Pull request merged**
